src/CommunityToolkit.Aspire.Hosting.NodeJS.Extensions/README.md

```diff
@@ -17,10 +17,10 @@ dotnet add package CommunityToolkit.Aspire.Hosting.NodeJS.Extensions
 Then, in the _Program.cs_ file of `AppHost`, define a Node.js resource, then call `AddYarnApp` or `AddPnpmApp`:
 
 ```csharp
-builder.AddYarnApp("yarn-demo")
+builder.AddYarnApp("yarn-demo", "../yarn-demo")
```

Comment:
Would it be possible to build `AddYarnApp` on top of `AddJavaScriptApp`?

Reply:
that's just legacy docs I have to clean up still, it's meant to be replaced by `AddJavaScriptApp().WithYarn()`